.travis.yml

```diff
@@ -25,7 +25,6 @@ script:
   - bundle exec rake docs:build
 
 rvm:
-  - jruby-9.1.12.0
```

Comment:
You need to restore this entry I think. Otherwise it won't run any jruby build, indeed.